index.html

```diff
@@ -28,26 +28,34 @@
   <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@600&display=swap" rel="stylesheet">
 </head>
 <body>
-  <div class="row head-container column-full-header">
+  <div class="row background-header height-header align-header column-full">
     <header>
       Code Journal
     </header>
   </div>
   <main>
-    <div class="row main-container" data-view="entry-form">
+    <div class="row container" data-view="entry-form">
```

Comment:
One element cannot be both a row and a container. You should give this container a child with the row class.